### PR TITLE
Replace hardcoded epsagon integration with a plugins mechanism

### PR DESCRIPTION
**Pull request by tripodsan**
should probably be  breaking change, since the azure adapter was also removed

- see #105 
- https://github.com/adobe/helix-epsagon/pull/257
- https://github.com/adobe/helix-slack-bot/pull/343

### CODE REVIEW

**Review by trieloff — APPROVED** [review]

File: src/aws-secrets.js [hunk 1]
```diff
@@ -49,19 +49,40 @@ async function loadAWSSecrets(functionName) {
...
   const now = Date.now();
-  if (!cache.data || now > cache.expiration) {
+  if (!cache.data || now > cache.date + expiration) {
     const params = await loadAWSSecrets(functionName);
     const nower = Date.now();
     // eslint-disable-next-line no-console
     console.info(`loaded ${Object.entries(params).length} package parameter in ${nower - now}ms`);
-    if (params) {
-      cache.data = params;
-      cache.expiration = nower + CACHE_EXPIRATION;
```
Comment (dominique-pfister) [inline src/aws-secrets.js:61]: hm, `nower` is in milliseconds, but `CACHE_EXPIRATION` in seconds? Is there a factor 1000 missing?

File: test/openwhisk-adapter.test.js [hunk 2]
```diff
@@ -40,7 +48,7 @@ describe('OpenWhisk Adapter Test', () => {
           });
           return new Response(ret);
         },
-        '@noCallThru': true,
+
```
Comment (dominique-pfister) [inline test/openwhisk-adapter.test.js:51]: Didn't know that 😄 Was it removed everywhere because it's unnecessary?

File: src/aws-secrets.js (hunk 1 above)
Comment (tripodsan) [reply src/aws-secrets.js:61]: oh....good catch!

File: test/openwhisk-adapter.test.js (hunk 2 above)
Comment (tripodsan) [reply test/openwhisk-adapter.test.js:51]: it's defined above already in the import:
```
const proxyquire = require('proxyquire').noCallThru();
```

**Pull request merged**